packages/grid/src/GridTheme.ts

```diff
@@ -146,6 +147,7 @@ const defaultTheme: GridTheme = Object.freeze({
   backgroundColor: '#000000',
   black: '#000000',
   white: '#ffffff',
+  primary: '#4878ea',
```

Comment:
We want to define it as a specific usage for theming, and not just a general color. 

I would put it adjacent to `textColor` defined as something like `textHyperlinkColor`, and then update the other references to match.

Then in irisgridtheme.module.scss that's where you define theme that the color should be $primary (so just some naming needs to be updated to match on that side).